packages/schema/index.ts

```diff
@@ -51,6 +51,37 @@ export const PoolMetadataSchema = z
   .array(MetadataItemSchema)
   .describe("My neat object schema");
 
+export const getInternalBalanceSchema = (totalBalance: number) => {
+  const InternalBalanceSchema = z.object({
+    tokenAddress: z.string().min(1),
```

Comment:
you could validate the address as well, it must be a 42 characters string starting with 0x.